### ISSUE TITLE
Changes to TNRIS.org after launch of data site.

### ISSUE THREAD

**Issue by mitchellryant** [TNRIS/tnris.org#1702]
Here is running list of changes to tnris.org that will need to be ready at launch of new data site. Please feel free to add to list and offer suggestion on how work can be completed. 

1. - [ ] Change top nav.  replace Maps and Data with Data and link to new site. @christoball 
2. - [ ] Remove Maps and Data page @christoball 
3. - [ ] Adjust footer to match top nav  @christoball 
4. - [ ] Replace 4 buttons on side of banner with new data site image. @christoball 
5. - [ ] Redirect all /data-download and all /data-catalog urls automatically to data.tnris.org @jwhaney @JasonKleinert @adambreznicky 
6. - [ ] Comment out all old data catalog entries stratmap pages @christoball @ThorTheDestroyer @gmullins @ilyannakadich  
7. - [ ] Comment out all old data catalog entries on old news article. @christoball @ThorTheDestroyer @gmullins @ilyannakadich  
8. - [ ] Remove old order forms and contact forms. only remaining should be a general inquiry to tnris form and the order custom maps form.
9. - [ ] Remove all buttons/links to /data-download and /data-catalog urls 

“Help will always be given at TNIRS to those who ask for it.”

**Comment by KCMack** [comment]
Help will always be given at TNIRS to those who ask for it.”
Make that TNRIS.

**Comment by mitchellryant** [comment]
3. Mockup
<img width="1112" alt="Footer" src="https://user-images.githubusercontent.com/8459928/54200647-e2f67b00-4499-11e9-82bb-61bb2f3e8f0d.png">

**Comment by KCMack** [comment]
Ryan,

I'll be in the office later today and I'll ask you to describe what I can do to help in this effort.

Thanks very much,
Kathleen

Kathleen C. Mack, PMP
Operations Manager
Texas Natural Resources Information System
1700 N. Congress Avenue
Austin, Texas 78701
Phone 512 463-4536/Cell 512 415-3847
Kathleen.Mack@twdb.texas.gov

Direct: (512) 463-4536
Main: (512) 463-8337
________________________________

**Comment by mitchellryant** [comment]
For 6 & 7 discussed this with @gmullins. She agreed commenting out is best solution. Might need to re-write some of the StratMap Ortho Page.

**Comment by KCMack** [comment]
Thanks, Ryan.

I'm happy to craft a story about the new site with you.  You can provide the critical details important for people to know about, and I can wrap a lot of words around the message.

Thanks very much,
Kathleen

Kathleen C. Mack, PMP
Operations Manager
Texas Natural Resources Information System
1700 N. Congress Avenue
Austin, Texas 78701
Phone 512 463-4536/Cell 512 415-3847
Kathleen.Mack@twdb.texas.gov

Direct: (512) 463-4536
Main: (512) 463-8337
________________________________

**Comment by mitchellryant** [comment]
for 10 and 13.

I have an idea of creating tutorial videos to help.

**Comment by christoball** [comment]
@mitchellryant just double checking we're scrapping the online mapping services page too, right? 

https://tnris.org/maps-and-data/online-mapping-services/

**Comment by mitchellryant** [comment]
@christoball yes that will be removed too. all that info is on the collection cards.

**Comment by mitchellryant** [comment]
Since we are going with a soft launch lets push the publication of the of the banner and news article till mid week. It will give me some time to develop the news article a bit more. @JasonKleinert @adambreznicky @jwhaney 

Rest of the changes listed above will be pushed on Monday morning.

Sorry for the late notice, wanted to discuss this you earlier.
